Makefile.kokkos

```diff
@@ -12,11 +12,12 @@ KOKKOS_DEVICES ?= "Pthread"
 # AMD-CPUS: AMDAVX,Ryzen,EPYC
 KOKKOS_ARCH ?= ""
 # Options: yes,no
-KOKKOS_DEBUG ?= "no"
+KOKKOS_DEBUG ?= "yes"
 # Options: hwloc,librt,experimental_memkind
 KOKKOS_USE_TPLS ?= ""
 # Options: c++11,c++14,c++1y,c++17,c++1z,c++2a
-KOKKOS_CXX_STANDARD ?= "c++11"
+#KOKKOS_CXX_STANDARD ?= "c++11"
+KOKKOS_CXX_STANDARD ?= "c++14"
```

Comment:
Is there something that requires C++14?

Reply:
I think I just did that since Christian said they were migrating to c++14 'soon'. I should definitely try with 11.

Reply:
Only a small modification was needed to accomodate c++11.